frontend/src/actions/users/confirm_preapproval_key.js

```diff
@@ -0,0 +1,44 @@
+import { post } from '../../lib/api';
+import * as constants from '../../constants/users';
+
+/**
+ * Confirm approval key
+ */
+
```

Comment:
we usually don't add an extra space between the comment of a function and the function.